crates/ursa-network/src/service.rs

```diff
@@ -464,6 +479,21 @@ where
                 message_id,
                 message,
             } => {
+                if !self.peer_blacklist.contains(&propagation_source) {
```

Comment:
What about a unit test for the `peer_blacklist`? Is that being populated atm?

Reply:
No, it is not being populated right now. I considered it but I think that deciding when/why/for how long to blacklist a peer is worthy of a separate PR.

Reply:
In that case, it would be cleaner to not include the blacklisting code until it is used.